test/known_issues/test-fs-fchmod-12835.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+const common = require('../common');
+if (common.isWindows) {
```

Comment:
I'll need to set it to be excluded for every platform but windows.
This way seems less noisy...